modules/gdscript/gdscript_cache.cpp

```diff
@@ -183,20 +183,26 @@ Ref<GDScript> GDScriptCache::get_shallow_script(const String &p_path, const Stri
 	return script;
 }
 
-Ref<GDScript> GDScriptCache::get_full_script(const String &p_path, Error &r_error, const String &p_owner) {
+Ref<GDScript> GDScriptCache::get_full_script(const String &p_path, Error &r_error, const String &p_owner, const bool p_update_from_disk) {
 	MutexLock lock(singleton->lock);
 
 	if (!p_owner.is_empty()) {
 		singleton->dependencies[p_owner].insert(p_path);
 	}
 
+	Ref<GDScript> script;
 	r_error = OK;
 	if (singleton->full_gdscript_cache.has(p_path)) {
-		return singleton->full_gdscript_cache[p_path];
+		script = (Ref<GDScript>)(singleton->full_gdscript_cache[p_path]);
```

Comment:
We normally don't use C style casting for Ref, instead it should be `Ref<T>(...)`.

Reply:
OK, thanks for pointing it out.